eqsl.yaml.sample

```diff
@@ -5,19 +5,14 @@ myrig: Icom IC7200
 ituzone: 6
 cqzone: 3
 
-# adif_file: /tmp/export.adif
+qsl_cache: "./cache"
+adif_file: "/path/to/your/wsjtx_log.adi"
 
 # Some logging software do not export the email address in the ADIF file.
 # eqsl will try to fetch the email address from qrz.com. For that to work
 # you need to provide a qrz API key.
 # The key can be found on qrz -> logbook -> settings -> API key
-qrzr_key: 123-abc-456-def
-
-# Some logging software do not export the email address in the ADIF file.
-# eqsl will try to fetch the email address from qrz.com. For that to work
-# you need to provide a qrz API key.
-# The key can be found on qrz -> logbook -> settings -> API key
-qrzr_key: 123-abc-456-def
+qrz_key: "your-qrz-api-key-here"
```

Comment:
I just noticed that this must actually be your password, the qrz API didn't really work, can you confirm @0x9900 ?